ORK1Kit/ORK1Kit/Common/ORK1FormStepViewController.m

```diff
@@ -171,6 +171,7 @@ - (void)addFormItem:(ORK1FormItem *)item {
         
         [textChoiceAnswerFormat.textChoices enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
             ORK1TableCellItem *cellItem = [[ORK1TableCellItem alloc] initWithFormItem:item choiceIndex:idx];
+            _cellItemForFormItem[item] = cellItem;
```

Comment:
This change does not appear in ORKFormStepViewController.m. Is that ok?

Reply:
Yes, that should be there - good catch! Will think if I can add a test later to capture this.